docker/docker-compose.yml

```
@@ -69,6 +69,7 @@ services:
       REACT_APP_API_URL: ${API_ROOT_URL}
       REACT_APP_ENVIRONMENT: ${NODE_ENV}
       REACT_APP_WIDGET_EMBED_PATH: ${WIDGET_EMBED_PATH}
+      REACT_APP_GITHUB_AUTH_OPTION: ${REACT_APP_GITHUB_AUTH_OPTION}
```

```suggestion
      REACT_APP_DOCKER_HOSTED_ENV: true
```

Hi @BiswaViraj a small suggestion, let me know what you think.

The reasoning would be that we might later add some more conditional functionality based on whether the user is in a docker mode or not, so we can use a more general static name that cannot be changed on the env. (We can even consider adding this to the docker file itself).

---

Yes, it makes sense to update it to a generic one.
I'll update the PR